packages/test-snaps/src/features/snaps/state/constants.ts

```diff
@@ -0,0 +1,5 @@
+import packageJson from '@metamask/manage-state-example-snap/package.json';
+
+export const MANAGE_STATE_SNAP_ID = 'npm:@metamask/manage-state-example-snap';
```

Comment:
Can't this be part of the existing feature in test-snaps?

Reply:
It could, but it got quite messy, so I ended up moving it out.

Reply:
I find the split a bit messy right now too, wondering what we could do to improve it

Reply:
Is there no path to refactor this to be the same "feature" on the test-snaps site?

Reply:
I mean I could just put it in the same card, but it would be very long and look weird.

Reply:
Okay, can we at least mark that the old one is "legacy" and this is the new API or something?

Is there any of the code that we can re-use across the cards?